### DISCUSSION TITLE
How can I "goto file" from the test summary window?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [nvim-neotest/neotest#260] [Q&A] [answered]
I'd like to be able to run all tests in the test suite and for the failing ones, go to the summary window and hit `gf` or similar, to go to the failing test, so I can inspect it. But I can't figure out how to go to the selected test.

I have read the docs and searched the discussions, and I feel this is likely possible - but I can't find a way to do it. The apparent `gf` doesn't work.

Is it possible to do this?

**Comment by fredrikaverpil** [comment, marked as answer]
Solved it. You can hit `i` to jump to a test from the summary. To see all the keymaps that exist in the summary:

1. Open the test summary window and move to it.
1. Run `:Telescope keymaps`
2. Search for `neotest`.

![image](https://github.com/nvim-neotest/neotest/assets/994357/08165908-a6f1-451b-ac04-bb8a8f6fd659)

**Reply by rcarriga** [reply]
You can also check `:h neotest.Config.summary.mappings` and `neotest.setup()` to see the defaults and explanations :smile:
